pkg/config/config.go

```diff
@@ -141,6 +141,10 @@ type RootConfig struct {
 	// explicitly handled by other options will be stored.
 	RunRoot string `toml:"runroot"`
 
+	// ImageStore if set it will allow end-users to store newly pulled image
+	// in path provided by `ImageStore` instead of path provided in `Root`.
+	ImageStore string `toml:"imagestore"`
```

Comment:
Do we need a CLI parameter for that?

Reply:
Podman has one, but I am not quite sure about how frequently `cri-o` users use the cli flags, but yeah its upto `cri-o` maintainers to decided on that. I'll add it if maintainers are okay with it :)

Reply:
i think we probably do want the cli flag to match --root flag

Reply:
@haircommander Do you prefer that in this PR or a different PR  ?  ( I'd like to open a new PR for frontend with a test for it so it is easier to review )

Reply:
i'd prefer this if possible

Reply:
Done.